From 58103554de82a24b55b79e8319027b19c66f7e14 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Sat, 18 Apr 2026 09:09:04 +0100
Subject: [PATCH] docs: -i disk allows multiple disk images

Commit 255722cbf3 ("v2v: Modular virt-v2v") which modularized the
-i disk code silently added the ability to read from multiple disks.
The documentation was not updated and still stated that only a single
disk could be used.

Remove incorrect statement in the documentation.

Fixes: commit 255722cbf39afc0b012e2ac00d16fa6ba2f8c21f
(cherry picked from commit 9b758f808710490ccdb4c35d58b0861b32d295d8)
---
 docs/virt-v2v.pod | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/docs/virt-v2v.pod b/docs/virt-v2v.pod
index 945e8dfc..63d5814e 100644
--- a/docs/virt-v2v.pod
+++ b/docs/virt-v2v.pod
@@ -273,8 +273,7 @@ Set the input method to I<disk>.
 In this mode you can read a virtual machine disk image with no
 metadata.  virt-v2v tries to guess the best default metadata.  This is
 usually adequate but you can get finer control (eg. of memory and
-vCPUs) by using I<-i libvirtxml> instead.  Only guests that use a single
-disk can be imported this way.
+vCPUs) by using I<-i libvirtxml> instead.
 
 =item B<-i> B<libvirt> name|UUID
 
